client/cmd/job/delete.go

```diff
@@ -0,0 +1,139 @@
+package job
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/goto/salt/log"
+	"github.com/spf13/cobra"
+
+	"github.com/goto/optimus/client/cmd/internal/connection"
+	"github.com/goto/optimus/client/cmd/internal/logger"
+	"github.com/goto/optimus/client/cmd/internal/survey"
+	"github.com/goto/optimus/config"
+	pb "github.com/goto/optimus/protos/gotocompany/optimus/core/v1beta1"
+)
+
+const (
+	deleteTimeout = time.Minute * 5
+)
+
+type deleteCommand struct {
+	deleteSurvey    *survey.JobDeleteSurvey
+	namespaceSurvey *survey.NamespaceSurvey
+	logger          log.Logger
+
+	connection connection.Connection
+
+	configFilePath string
+	force          bool
+	cleanHistory   bool
+
+	clientConfig *config.ClientConfig
+}
+
+// NewDeleteCommand initializes job delete command
+func NewDeleteCommand() *cobra.Command {
+	l := logger.NewClientLogger()
+	deleteCmd := &deleteCommand{
+		deleteSurvey:    survey.NewJobDeleteSurvey(),
+		namespaceSurvey: survey.NewNamespaceSurvey(l),
+		logger:          l,
+	}
+
+	cmd := &cobra.Command{
+		Use:     "delete <job_name>",
+		Short:   "Delete an existing job in the server",
+		Example: "optimus job delete daily_scheduled_job",
```

Comment:
my only concern is the scope of this command is different from `job create`, since `create` is just creating the spec in the repository, afraid `delete` will be expected to delete the spec in the repository.

Reply:
yes, that could also be put into consideration. but, in the description, it says something like:
```go
"Delete an existing job in the server"`
```

won't it be enough? If by giving this description it's already enough, then perhaps we can also update the one in `create` as there it only describes it like:

```go
"Create a new Job"
```

into

```go
"Create a new job specification in local"
```

Reply:
as being discussed through offline, updated description for job create to make it more explicit